### ISSUE TITLE
Endpoints to support retrieving room and building numbers

### ISSUE THREAD

**Issue by kippmr** [kippmr/pinpointr#50]
According to new flow chart some new communication between backend and app is required

1. app sends image and receives tags
1. app sends location data and receives room and building number
1. whatever is missing the app must require user to enter this information

**Comment by kippmr** [comment]
@mckaysm make sure this rough outline in the comment above is accurate please

**Comment by mckaysm** [comment]
Yes, that looks correct. I'll copy the flowchart I drew to a doc shortly

**Comment by kippmr** [comment]
When you do can you add it to the wiki?

**Comment by mckaysm** [comment]
Yep for sure

**Comment by mckaysm** [comment]
Added to wiki
